style.css

```diff
@@ -0,0 +1,314 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  padding: 0 50px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template: repeat(6, minmax(150px, 1fr)) / repeat(6, minmax(150px, 1fr));
+  grid-template-areas: "sidebar header header header header header" "sidebar content content content content content" "sidebar content  content content content content" "sidebar content content content content content " "sidebar content content content content content ";
+  height: 100vh;
+}
+
+#header {
+  padding: 0px 40px;
+  grid-area: header;
+}
+
+#side-bar {
+  grid-area: sidebar;
+  background-color: #0d83dd;
+  font-size: larger;
+}
+
+#content {
+  grid-area: content;
+  display: flex;
+  flex-direction: column;
+  background-color: #f1f3f6;
+  padding: 20px;
+}
+
+.topic {
+  padding: 5px 0 5px 0;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+}
+.card-container {
```

Comment:
great work!
It is always advisable to leave an empty line between each element selector , or group of properties.
to make it easier, you may consider using css linters checks or some css formatter plugins on your favourite editor.